augraphy/augmentations/inkbleed.py

```diff
@@ -18,6 +19,10 @@ class InkBleed(Augmentation):
     :param color_range: Pair of ints determining the range from which color
            noise is sampled.
     :type color_range: tuple, optional
+    :param kernel_size: Kernel size to determine area of inkbleed effect.
+    :type kernel_size: tuple, optional
+    :param severity: Severity to determine concentration of inkbleed effect.
```

Comment:
I also added severity, to allow user adjust the concentration inkbleed effect.

For example, severity = (0.1, 0.2) and (0.8,0.9):
![image](https://user-images.githubusercontent.com/52521979/147101992-c7c8d54c-61fc-45f4-95fd-ff0a3bfe28f0.png)